IdentityCore/src/webview/embeddedWebview/MSIDOAuth2EmbeddedWebviewController.m

```diff
@@ -174,6 +174,10 @@ - (BOOL)loadView:(NSError **)error
 - (void)endWebAuthWithURL:(NSURL *)endURL
                     error:(NSError *)error
 {
+    if (self.complete)
+    {
+        return;
```

Comment:
can we add an info level log here for diagnostics?

Reply:
Added.